Toniof enhance dataset descriptor (#402)
dcs4cop/xcube: CHANGES.md

## Changes in 0.6.2 (in development)

> * Added `coords` parameter to DatasetDescriptor. 
>   Also, show a DatasetDescriptor' data variables as dictionary. 

Reviewer: ```suggestion
* Added `coords` property to `DatasetDescriptor` class. 
  The `data_vars` property of the `DatasetDescriptor` class is now dictionary. 
```
